### DISCUSSION TITLE
Welcome to Ocelot Discussions!

### DISCUSSION THREAD

**Discussion by TomPallister** [ThreeMammals/Ocelot#1723] [Announcements]
## 👋 Welcome! :tiger:
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by raman-m** [comment]
Hello, Ocelot community!

I am the repository maintainer and project coordinator. @TomPallister has not led the project since 2020. However, he remains invested and continues to congratulate us on new releases.

### Ocelot development team
- @raman-m is available for communication every day
- @ggnaegi is available for communication every working day
- @RaynaldM is **sometimes** available for communication!

Welcome to Ocelot world! :tiger:
